ztests/suite/zqd/rec-curl.yaml

```diff
@@ -0,0 +1,84 @@
+script: |
+  source recserv.sh
+  echo ===
+  curl --header "Content-Type: application/json" -request POST \
+    --data '{"addr":"a.b.c:5000","node":"a.b"}' \
+    http://$ZQD_HOST/register 2> err
```

Comment:
Though I certainly respect the simplicity of the api, all of our requests have supporting zapi commands. I think we should add recruiter commands to the connection api and add recruiter commands to zapi, then execute these commands using zapi. This would make these test more concise and readable.

Reply:
Yes, my current working branch implements the changes to the connection api. However, I think it is important that we test these commands with curl also, since there are likely to be situations in K8s deployment where a system other that our client code will need to call one or or more of these API's.